examples/js/loaders/BasisTextureLoader.js

```diff
@@ -22,6 +20,7 @@ THREE.BasisTextureLoader = function ( manager ) {
 
 	THREE.Loader.call( this, manager );
 
+	this.useAlpha = true;
```

Comment:
Do we need this option? We can detect that the file has alpha and choose the right format that way, right?

If it's necessary, the option would need to be part of the task configuration rather than the loader configuration. A single loader should be able to transcode images with and without alpha in parallel, without downloading another copy of the transcoder. For example:

```js
const map = loader.loadRGB( 'foo.basis', { ...options } );
const alphaMap = loader.loadAlpha( 'foo.basis', { ...options } );
```

^That's likely to be a hassle, as the texture can't be transferred to two workers at once without copying, so I'm hoping we can avoid it for now.

Reply:
My understanding is the use case should be able to choose use alpha channel or not regardless of existence of alpha channel in basis texture. As some compressed format (PVRTC) with alpha has fewer bits for RGB channels. It's possible to use a compressed format without alpha channel when transparency is not needed. What's your thought?

Reply:
Maybe I will use auto-detected alpha for now. If there's any issue brought up by devs we could then try the refactor